### PR TITLE
React

### PR DESCRIPTION
**Pull request by iefserge**
T = major

### CODE REVIEW

File: lib/embeds.js [hunk 1]
```diff
@@ -11,63 +9,112 @@ function renderImage (props) {
     return '';
   }
 
-  return <amp-img width={width} height={height} layout='responsive' src={src}></amp-img>;
+  return <amp-img width={width} height={height} layout='responsive' src={src} />;
 }
 
-function renderYoutube (props) {
+renderImage.propTypes = {
+  width: React.PropTypes.number,
```
Comment (ellell) [inline lib/embeds.js]: Width and height are required here right?

**Comment by ellell** [comment]
This looks great, just one minor comment!

File: lib/embeds.js (hunk 1 above)
Comment (iefserge) [inline lib/embeds.js]: yeah

Comment (iefserge) [inline lib/embeds.js]: @ellell sorry, actually, no. It should remove images that don't have width or height.

**Pull request merged**